### PR TITLE
fix: Color group support fixes in transparent header for new customizer

### PR DESCRIPTION
**Pull request by abhishek-bsf**
### Description

Color group support fixes in transparent header for new customizer

### Types of changes
Bug fix (non-breaking change which fixes an issue)

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests

### CODE REVIEW

File: inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php [hunk 1]
```diff
@@ -610,56 +609,43 @@ public function register_configuration( $configurations, $wp_customize ) {
...
 					array(
-						'name'              => 'transparent-header-search-box-background-color',
+						'name'              => ASTRA_THEME_SETTINGS . '[transparent-header-search-box-background-color]',
 						'default'           => astra_get_option( 'transparent-header-search-box-background-color' ),
-						'parent'            => ASTRA_THEME_SETTINGS . '[transparent-header-search-colors-group]',
-						'type'              => 'sub-control',
+						'type'              => 'control',
 						'section'           => 'section-transparent-header',
-						'priority'          => 6,
+						'priority'          => 45,
```
Comment (vrundakansara) [inline inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php:642]: Please confirm this with old and new header. Check if changing the priority doesn't mess up the sequence.

File: inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php [hunk 2]
```diff
@@ -610,56 +609,43 @@ public function register_configuration( $configurations, $wp_customize ) {
...
 						'transport'         => 'postMessage',
 						'control'           => 'ast-color',
 						'sanitize_callback' => array( 'Astra_Customizer_Sanitizes', 'sanitize_alpha_color' ),
-						'title'             => __( 'Box Background Color', 'astra' ),
+						'title'             => __( 'Secrch Box Background', 'astra' ),
```
Comment (vrundakansara) [inline inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php:646]: Typo in the text `Search Box Background`

File: inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php [hunk 3]
```diff
@@ -695,34 +670,30 @@ public function register_configuration( $configurations, $wp_customize ) {
...
 						'sanitize_callback' => array( 'Astra_Customizer_Sanitizes', 'sanitize_alpha_color' ),
 						'section'           => 'section-transparent-header',
 						'transport'         => 'postMessage',
-						'priority'          => 10,
-						'tab'               => __( 'Normal', 'astra' ),
-						'title'             => __( 'Content Color', 'astra' ),
-						'context'           => Astra_Builder_Helper::$general_tab,
+						'priority'          => 49,
+						'title'             => __( 'Widget Content', 'astra' ),
+						'context'           => Astra_Builder_Helper::$design_tab,
```
Comment (vrundakansara) [inline inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php:696]: Have we also changed the general tab to design tab?

Comment (abhishek-bsf) [reply inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php:696]: This control was previously a sub-control whose parent control context was `Astra_Builder_Helper::$design_tab`. We transformed it as a main control and made the context similar to parent control context.

File: inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php (hunk 1 above)
Comment (abhishek-bsf) [reply inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php:642]: Updated priority and checked sequence in old as well as new customizer.

File: inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php (hunk 2 above)
Comment (abhishek-bsf) [reply inc/addons/transparent-header/classes/sections/class-astra-customizer-transparent-header-configs.php]: Corrected the string.

**Review by vrundakansara — APPROVED** [review]

**Pull request merged**